netbox/templates/extras/htmx/script_result.html

```diff
@@ -41,7 +41,7 @@ <h5 class="card-header">{% trans "Test Summary" %}</h5>
     <div class="card">
       <div class="table-responsive" id="object_list">
         <h5 class="card-header">{% trans "Log" %}</h5>
-        {% include 'htmx/table.html' %}
+        {% include 'htmx/table.html' with no_htmx_container=True%}
```

Comment:
I don't think this is a pattern we want to encourage. I'm going to see if I can come up with a cleaner fix for this.